.gitignore

```diff
@@ -158,3 +158,4 @@ fabric.properties
 .LSOverride
 
 
+*.checkstyle
```

Comment:
I installed a plugin on eclipse that check style in real time, it generated a .checkstyle file in the directory, so I added .checkstyle files to gitignore